TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/PageContextFeatures.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.AspNetCore.Http.Features;
+using TeachingRecordSystem.Core.DataStore.Postgres.Models;
+using TeachingRecordSystem.Core.Dqt.Models;
+
+namespace TeachingRecordSystem.SupportUi;
+
+public static class HttpContextExtensions
+{
+    public static CurrentPersonFeature GetCurrentPersonFeature(this HttpContext context) =>
+        context.Features.GetRequiredFeature<CurrentPersonFeature>();
+
+    public static void SetCurrentPersonInfoFeature(this HttpContext context, CurrentPersonFeature currentPersonInfo) =>
```

Comment:
maybe this should not contain the word "Info" for consistency i.e. SetCurrentPersonFeature

Reply:
Good catch, fixed.